tests/integration/Sample.spec.ts

```diff
@@ -0,0 +1,60 @@
+import "expect-puppeteer";
```

Comment:
As you can see from the run that failed in CI, the individual `it` statements depend on each other. This entire workflow should really be separated into a variety of tests/files around 1) Register 2) Sign in 3) Using the samples 4) uploading own files